catalog/weather/forecast.js

```diff
@@ -52,10 +51,13 @@ function main(params) {
             console.log('http status code:', response.statusCode);
             console.log('error:', error);
             console.log('body:', body);
-            whisk.error(body || error);
+	        whisk.error({
```

Comment:
do you mean to double nest the `error` property? or do you mean to use `whisk.done` instead?